src/game/puppyprint.c

```diff
@@ -421,7 +433,7 @@ void append_puppyprint_log(const char *str, ...) {
 #ifdef UNF
     osSyncPrintf(textBytes);
```

Comment:
this could use a bonus newline but nbd

Reply:
for reference, not the code but the string cus otherwise itll clog the console up